src/sidebar/store/test/index-test.js

```diff
@@ -289,15 +289,6 @@ describe('store', function () {
     ]);
   });
 
-  describe('#subscribe()', function () {
```

Comment:
I'm not seeing where subscribe got moved to in the new implementation, also not seeing any tests for this behavior?

Reply:
`subscribe` is actually part of the basic [store API](https://redux.js.org/api-reference/store#subscribe). However I added an additional test (`"notifies subscribers when state changes"`) in `create-store.js`.

Most of the tests that are currently in `index-test.js` will be moved to `store/modules/{relevant module}-test.js` but after that I think it will make sense to have some integration tests to verify that all the pieces work together as expected.